From 868ee948b626dced91bb98207ebd6b3a52f9fcfc Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Fri, 3 Jul 2020 13:28:39 -0400
Subject: [PATCH 2/7] rt-tests: pi_stress.8: Remove unused -t n from the
 manpage

The -t option was replaced with the -D, --duration=TIME option
Remove it from the manpage as well.

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 src/pi_tests/pi_stress.8 | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/pi_tests/pi_stress.8 b/src/pi_tests/pi_stress.8
index 5e2af6189ca1..ed03ad6187e7 100644
--- a/src/pi_tests/pi_stress.8
+++ b/src/pi_tests/pi_stress.8
@@ -15,8 +15,6 @@ pi_stress \- a stress test for POSIX Priority Inheritance mutexes
 .B pi_stress
 .RB [ \-i|\-\-inversions
 .IR inversions ]
-.RB [ \-t|\-\-duration
-.IR seconds ]
 .RB [ \-g|\-\-groups
 .IR groups
 .RB [ \-d|\-\-debug ]
@@ -50,10 +48,6 @@ Run for
 .I n
 number of inversion conditions. This is the total number of inversions
 for all inversion groups. Default is \-1 for infinite.
-.IP "\-t n|\-\-duration=n"
-Run the test for 
-.I n
-seconds and then terminate.
 .IP "\-g n|\-\-groups=n"
 The number of inversion groups to run. Defaults to 10.
 .IP \-d|\-\-debug
-- 
2.21.3

